### PR TITLE
User Story 1 - Landing page

### PR DESCRIPTION
**Pull request by kevinldg**
Landing page with a list of 4 upcycling ideas for a general overview.

Corresponding user story: https://github.com/kevinldg/upcycling-app/issues/1

### CODE REVIEW

File: components/Card/index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+`;
+
+export default function Card({ image, title, hashtags }) {
+  return (
+    <Article>
+      <StyledImage src={image} alt={title} width={150} height={120} />
+      <h2>{title}</h2>
+      <Hashtags>
+        {hashtags.map((hashtag, index) => (
+          <li key={index}>#{hashtag}</li>
```
Comment (CJAkkuec) [inline components/Card/index.js:28]: Please don't use the index of an array to provide `key` values. I'm assuming that the hashtags may change at some point, which is why the index doesn't suffice to keep track of the list items.

File: pages/index.js [hunk 2]
```diff
@@ -1,7 +1,49 @@
+console.clear();
```
Comment (CJAkkuec) [inline pages/index.js:1]:
```suggestion
```

File: pages/index.js [hunk 3]
```diff
@@ -1,7 +1,49 @@
...
+
+import { useState } from "react";
+import styled from "styled-components";
+
+import { ideas } from "@/lib/db";
+import Card from "../components/Card";
+import Header from "@/components/Header";
+
+const defaultIdeas = ideas;
```
Comment (CJAkkuec) [inline pages/index.js:10]:
```suggestion
```

File: pages/index.js [hunk 4]
```diff
@@ -1,7 +1,49 @@
+console.clear();
+
+import { useState } from "react";
+import styled from "styled-components";
+
+import { ideas } from "@/lib/db";
```
Comment (CJAkkuec) [inline pages/index.js:6]:
```suggestion
import { ideas as defaultIdeas } from "@/lib/db";
```

File: pages/index.js [hunk 5]
```diff
@@ -1,7 +1,49 @@
...
+import Card from "../components/Card";
+import Header from "@/components/Header";
+
+const defaultIdeas = ideas;
+
+const CardContainer = styled.ul`
```
Comment (CJAkkuec) [inline pages/index.js:12]:
```suggestion
const CardList = styled.ul`
```
If you're working with ListItem as a suffix, then I'd rename this to CardList.

File: pages/index.js [hunk 6]
```diff
@@ -1,7 +1,49 @@
...
+  flex-wrap: wrap;
+  gap: 1rem;
+  list-style: none;
+  padding: 0;
+  margin-top: 1rem;
+`;
+
+const CardListItem = styled.li`
+  flex: 0 0 calc(50% - 1rem);
+  max-width: calc(50% - 1rem);
```
Comment (CJAkkuec) [inline pages/index.js:23]: I think that's going to be pretty small on a mobile screen. 🤔

File: styles.js [hunk 7]
```diff
@@ -5,10 +5,24 @@ export default createGlobalStyle`
...
 
   body {
-    margin: 0;
     font-family: system-ui;
+    margin: 1rem 0rem 0rem 1rem;
+  }
+
+  header {
+    margin-bottom: 2rem;
+  }
```
Comment (CJAkkuec) [inline styles.js:19]: Please move the styling to the header component. 🙏 

Please do this for the `ul` and `h1` selector as well.

File: styles.js [hunk 8]
```diff
@@ -5,10 +5,24 @@ export default createGlobalStyle`
...
   *::after {
     box-sizing: border-box;
+    margin: 0;
+    padding: 0;
   }
 
   body {
-    margin: 0;
     font-family: system-ui;
+    margin: 1rem 0rem 0rem 1rem;
```
Comment (CJAkkuec) [inline styles.js:14]: I'm not really sure why you're "indenting" the body by 1rem top and left, but maybe there's a better way to layout your content?

**Review by CJAkkuec — CHANGES_REQUESTED** [review]
Yaaaay! Your first Capstone PR and code review! 🎉 

Looking good overall, there's just a few small things that need to be addressed.

**Comment by kevinldg** [comment]
Thx @CJAkkuec for your feedback and suggested changes. We have implemented them 👍

**Review by CJAkkuec — APPROVED** [review]
Nice job! Approved! 🚀

(Hopefully the merge conflict is a small one. 🙈 )

**Pull request merged**